test/Microsoft.Azure.ServiceBus.UnitTests/SenderReceiverClientTestBase.cs

```diff
@@ -124,7 +124,7 @@ public abstract class SenderReceiverClientTestBase
             TestUtility.Log("Sleeping 10 seconds...");
             await Task.Delay(TimeSpan.FromSeconds(10));
 
-            DateTime lockedUntilUtcTime = await messageReceiver.RenewLockAsync(receivedMessages.First().LockToken);
+            var lockedUntilUtcTime = await messageReceiver.RenewLockAsync(receivedMessages.First().LockToken);
```

Comment:
@SeanFeldman, this particular instance might fall in the grey area because of the naming convention of the variable. And I guess this is one of those never ending coding practices debate. Personally, I'd prefer that from next set of PRs, we use `var` with discretion and not everytime ReSharper suggests it :) 

Regarding
> The codebase is no longer a few files. To actually read it and understand, a review of a diff on GH is not enough. It either requires to pull the code down and see it all, or requires to be familiar with the code. I'd say if you pull the code down, you will be using an IDE.

I neither pull down the code and do the review in IDE, nor do I claim to be an expect with the entire code...

And regarding
> I see `var` as engine

I'm just looking out for people who see it as `object` and not engine.